integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -233,73 +257,158 @@ class AuthorizationServerIT extends ITSupport {
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition)
+                    .setInclude(
+                        [OAuth2Scope.MetadataPublishEnum.ALL_CLIENTS.name()]
+                    )
+                )
+            )
+        AuthorizationServerPolicy createdPolicy = createdAuthorizationServer.createPolicy(authorizationServerPolicy)
         assertThat(createdPolicy, notNullValue())
 
-        createdAuthorizationServer.deletePolicy(createdPolicy.getId())
+        AuthorizationServerPolicy retrievedPolicy = createdAuthorizationServer.getPolicy(createdPolicy.getId())
+        assertThat(retrievedPolicy, notNullValue())
+        assertThat(retrievedPolicy.getId(), equalTo(createdPolicy.getId()))
 
-        // delete may not effect immediately in the backend
-        sleep(getTestOperationDelay())
+        AuthorizationServerPolicyRule createdPolicyRule = retrievedPolicy.createPolicyRule(createdAuthorizationServer.getId(),
+            client.instantiate(AuthorizationServerPolicyRule)
+                .setName(name)
+                .setType(AuthorizationServerPolicyRule.TypeEnum.ACCESS)
+                .setPriority(1)
+                .setConditions(client.instantiate(AuthorizationServerPolicyRuleConditions)
+                    .setPeople(client.instantiate(PolicyPeopleCondition)
+                        .setGroups(client.instantiate(GroupCondition)
+                            .setInclude(["EVERYONE"])))
+                    .setGrantTypes(client.instantiate(GrantTypePolicyRuleCondition)
+                        .setInclude(["implicit", "client_credentials", "authorization_code", "password"]))
+                    .setScopes(client.instantiate(OAuth2ScopesMediationPolicyRuleCondition).setInclude(["openid", "email", "address"]))
+                )
+        )
 
-        assertNotPresent(createdAuthorizationServer.listPolicies(), createdPolicy)
+        assertThat(createdPolicyRule, notNullValue())
+        assertThat(createdPolicyRule.getType(), equalTo(AuthorizationServerPolicyRule.TypeEnum.ACCESS))
+        assertThat(createdPolicyRule.getPriority(), equalTo(1))
+        assertThat(createdPolicyRule.getConditions().getPeople().getGroups().getInclude(), contains("EVERYONE"))
+        assertThat(createdPolicyRule.getConditions().getGrantTypes().getInclude(),
+            containsInAnyOrder("implicit", "client_credentials", "authorization_code", "password"))
+        assertThat(createdPolicyRule.getConditions().getScopes().getInclude(),
+            containsInAnyOrder("openid", "email", "address"))
+        assertThat(createdPolicyRule.getActions().getToken().getAccessTokenLifetimeMinutes(), equalTo(60))
```

Comment:
You're probably asserting on the default `actions` here (since you didn't explicitly set `actions` - see above comment).

Reply:
done